src/main/java/software/amazon/encryption/s3/internal/ConvertSDKRequests.java

```diff
@@ -13,7 +13,9 @@
 public class ConvertSDKRequests {
 
   public static PutObjectRequest convertRequest(CreateMultipartUploadRequest request) {
-
+    /*Converts a CreateMultipartUploadRequest into a PutObjectRequest by setting optional fields needed for
+    putInstructionFile operation.
+     */
```

Comment:
This should be a [Javadoc](https://docs.oracle.com/javase/8/docs/technotes/tools/windows/javadoc.html) comment. Meaning, it comes before the method and it has this syntax (most importantly, the two `*`s in the first block):

```
/**
 * Comment here
 */
public whatever methodName() {
```

There are many examples throughout the codebase. Your IDE should also render them as green rather than grey.

In terms of content, it is not really accurate. We should specify that the conversion is used when Instruction File PutObject is enabled and the customer performs a multipart upload. `putInstructionFile` doesn't exist in v3.